plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/revert/RevertCommitViewImpl.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.ide.ext.git.client.revert;
+
+import com.google.gwt.cell.client.TextCell;
+import com.google.gwt.core.shared.GWT;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.event.dom.client.ScrollEvent;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.uibinder.client.UiHandler;
+import com.google.gwt.user.cellview.client.CellTable;
+import com.google.gwt.user.cellview.client.Column;
+import com.google.gwt.user.client.ui.Button;
+import com.google.gwt.user.client.ui.ScrollPanel;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.gwt.view.client.SelectionChangeEvent;
+import com.google.gwt.view.client.SingleSelectionModel;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import java.util.List;
+import org.eclipse.che.api.git.shared.Revision;
+import org.eclipse.che.ide.ext.git.client.DateTimeFormatter;
+import org.eclipse.che.ide.ext.git.client.GitLocalizationConstant;
+import org.eclipse.che.ide.ext.git.client.GitResources;
+import org.eclipse.che.ide.ui.window.Window;
+
+/**
+ * The implementation of {@link RevertCommitView}.
+ *
+ * @author Dmitrii Bocharov (bdshadow)
+ */
+@Singleton
+public class RevertCommitViewImpl extends Window implements RevertCommitView {
+  interface RevertCommitViewImplUiBinder extends UiBinder<Widget, RevertCommitViewImpl> {}
+
+  private static RevertCommitViewImplUiBinder uiBinder =
+      GWT.create(RevertCommitViewImplUiBinder.class);
+
+  @UiField ScrollPanel revisionsPanel;
+
+  Button btnRevert;
+  Button btnCancel;
```

Comment:
Please make this variables private

Reply:
i agree about this one, however, once again, i wanted to make it consistent everywhere, that's why made as it is in `FetchViewImpl`, `AddRemoteRepositoryViewImpl` and others